### PR TITLE
AMQPHandler sends alerts to status_message.<level>.<service_name> whe…

### PR DESCRIPTION
**Pull request by guiguem**
…n level >=warning and !=35. Messages are posted according to a map defined in the Slack interface config file.

### CODE REVIEW

File: dragonfly/implementations/alert_spammer.py [hunk 1]
```diff
@@ -31,17 +31,17 @@ def __init__(self,broker=None,sleep_time = 10,*args, **kwargs):
...
+        self.level = 'warning'
         self.username = self.name
         self.sleep_time = sleep_time
 
     def spam(self):
         while (True):
 
-            severity = 'status_message.{}.{}'.format(self.this_channel,self.username)
+            severity = 'status_message.{}.{}'.format(self.level,self.username)
             print('sending to alerts exchange with severity {} message ({})'.format(severity,'Redundant informations'))
```
Comment (laroque) [inline dragonfly/implementations/alert_spammer.py]: should this be a logger.info()?

File: dragonfly/implementations/alert_spammer.py [hunk 2]
```diff
@@ -31,17 +31,17 @@ def __init__(self,broker=None,sleep_time = 10,*args, **kwargs):
...
             self.connection_to_alert.send_status_message(severity=severity,alert='Redundant informations')
-            logger.critical('I am critical')
+            logger.warning('I am critical')
```
Comment (laroque) [inline dragonfly/implementations/alert_spammer.py]: I am warning?

File: dragonfly/implementations/alert_spammer.py (hunk 1 above)
Comment (guiguem) [inline dragonfly/implementations/alert_spammer.py]: indeed

File: dragonfly/implementations/alert_spammer.py (hunk 2 above)
Comment (guiguem) [inline dragonfly/implementations/alert_spammer.py]: indeed too

File: dragonfly/status_log_handlers/amqp_handler.py [hunk 3]
```diff
@@ -42,16 +42,18 @@ def update_parser(self, parser):
 
     def setLevel(self, level):
         '''
-        for now, force the to critical...
+        for now, force the to warning...
         '''
-        if level != logging.CRITICAL:
-            print('warning: slack is only ever critical, setting that')
+        if level < logging.WARNING:
+            print('warning: slack is only ever warning, setting that')
```
Comment (laroque) [inline dragonfly/status_log_handlers/amqp_handler.py]: did you mean to also call `super(AMQPHandler, self).seLevel(logging.WARNING)` after this print?

Comment (guiguem) [inline dragonfly/status_log_handlers/amqp_handler.py]: done

**Pull request merged**